product_packaging_level_purchasable/models/product_packaging.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ProductPackaging(models.Model):
+    _inherit = "product.packaging"
+
+    purchases = fields.Boolean(
```

Comment:
The field name could be improved
```suggestion
    can_be_purchased = fields.Boolean(
```